backstop/tests/tabs.html

```diff
@@ -24,241 +24,919 @@
+                class="iui-tab-icon"
+                aria-hidden="true"
+                viewBox="0 0 16 16"
+              >
+                <path
+                  d="m0 0v16h16v-16zm14.85714 1.94286v12.11428l-6.05714-6.05714zm-6.85714 5.25714-6.05714-6.05714h12.11428zm-.8.8-6.05714 6.05714v-12.11428zm.8.8 6.05714 6.05714h-12.11428z"
+                />
+              </svg>
+              <span class="iui-tab-label">
+                <div>Left</div>
+                <div class="iui-tab-description">Sublabel</div>
+              </span>
+            </button>
+          </li>
+          <li>
+            <button
+              class="iui-tab"
+              role="tab"
+              aria-selected="false"
+              aria-controls="disabled-tab"
+              disabled
+            >
+              <!-- Use placeholder.svg from @itwin/itwinui-icons package. -->
+              <svg
+                class="iui-tab-icon"
+                aria-hidden="true"
+                viewBox="0 0 16 16"
+              >
+                <path
+                  d="m0 0v16h16v-16zm14.85714 1.94286v12.11428l-6.05714-6.05714zm-6.85714 5.25714-6.05714-6.05714h12.11428zm-.8.8-6.05714 6.05714v-12.11428zm.8.8 6.05714 6.05714h-12.11428z"
+                />
+              </svg>
+              <span class="iui-tab-label">
+                <div>Disabled</div>
+                <div class="iui-tab-description">Sublabel</div>
+              </span>
+            </button>
+          </li>
+          <li>
+            <button
+              class="iui-tab"
+              role="tab"
+              aria-selected="false"
+              aria-controls="none-tab"
+            >
+              <span class="iui-tab-label">None</span>
+            </button>
+          </li>
+        </ul>
+        <div class="iui-tab-stripe"></div>
+        <div
+          class="iui-tabs-content"
+          role="tabpanel"
+          id="active-tab"
+          aria-labelledby="active"
+        >
+          Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod
+          tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim
+          veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea
+          commodo consequat. Duis aute irure dolor in reprehenderit in voluptate
+          velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat
+          cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id
+          est laborum.
+        </div>
+      </div>
+
+    </section>
+
+    <script>
+      document.querySelectorAll('.iui-tabs:not(.iui-default)').forEach((tabs) => {
+        const wrapper = tabs.parentElement;
+        const activeTab = tabs.querySelector('.iui-active');
+        const stripe = wrapper.querySelector('.iui-tab-stripe');
+        const setStripe = (activeEl) => {
+          stripe.style.left = `${activeEl.offsetLeft}px`;
+          stripe.style.width = `${activeEl.offsetWidth}px`;
```

Comment:
These numbers not always match the real element:
![image](https://user-images.githubusercontent.com/36186912/122519265-f00c5800-d01a-11eb-8666-5240fe90a701.png)
As you can se on the right it goes outside the tab a bit. Because tab's width is `118.72px` so `offsetWidth` returns `119px`. We might need to use `getComputedStyle` or `getBoundingClientRect`.

Reply:
I changed width to use `getComputedStyle`, and it seemed better at first, but now I'm noticing different behavior on different zoom levels:

![image](https://user-images.githubusercontent.com/9084735/122767421-a452ec00-d270-11eb-9035-ec3832fc571c.png)
![image](https://user-images.githubusercontent.com/9084735/122767407-a026ce80-d270-11eb-9334-6acfd0043918.png)

Reply:
Changed to `getBoundingClientRect`. But then I was zooming in and out, and noticed that this JS approach doesn't work well for pill tabs because those are using flex (their width doesn't change).

Reply:
Yeah, we would need to observe resizes.